client/components/videos-ui/index.jsx

```diff
@@ -78,11 +77,6 @@ const VideosUi = ( { headerBar, onBackClick = () => {} } ) => {
 		}
 	};
 
-	const skipClickHandler = () =>
```

Comment:
This will need to be moved to the upcoming `onboarding-header-bar.jsx` component.